README.md

```diff
@@ -38,7 +38,7 @@
 - Write basic shell scripts to ease the programming life.
 
 
-## [JavaScript1](https://github.com/HackYourFuture/JavaScript/)/[GIT](https://github.com/HackYourFuture/Git)/ [debugging](https://github.com/HackYourFuture/debugging)
+## [JavaScript1](https://github.com/HackYourFuture/JavaScript1/)/[GIT](https://github.com/HackYourFuture/Git)/ [debugging](https://github.com/HackYourFuture/debugging)
 
```

Comment:
Are we actually still planning to do separate debugging hangouts? Is there any conclusion of the first attempts?

Reply:
The idea is to see if we can incorporate it in the javascript sessions, let's talk about it Sunday!